src/main.ts

```diff
@@ -1,9 +1,29 @@
+import { INestApplication } from "@nestjs/common";
 import { NestFactory } from "@nestjs/core";
+import { DocumentBuilder, SwaggerModule } from "@nestjs/swagger";
+import * as dotenv from "dotenv";
 
 import { AppModule } from "./app.module";
+import { Environment } from "./config/env.validation";
 
+dotenv.config();
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  if (process.env.NODE_ENV !== Environment.Production) {
```

Comment:
may not necessary? I don't think this docs need to be hidden when prod.

Reply:
I agree